docs/tutorials/comments.rst

```diff
@@ -35,7 +35,8 @@ entire URL:
 
 .. code-block:: python
 
-   submission = reddit.submission(url='https://www.reddit.com/r/funny/comments/3g1jfi/buttons/')
+   url='https://www.reddit.com/r/funny/comments/3g1jfi/buttons/'
```

Comment:
There's no need to break it up. It's okay if these lines span more than 80 characters.

Reply:
It does look better on smaller devices if they're smaller

Reply:
Optimizing our code for readability on smaller devices isn't really the goal.

Reply:
I don't see the harm. Sometimes if I'm busy I might use my phone. It isn't unreadable.

Reply:
```suggestion
   url = "https://www.reddit.com/r/funny/comments/3g1jfi/buttons/"
```